tests/numeric/test_InteractionTransformer.py

```diff
@@ -1,66 +1,25 @@
-import contextlib
-import re
-
 import numpy as np
 import pandas as pd
 import pytest
 import test_aide as ta
 
 import tests.test_data as d
-import tubular
+from tests.numeric.test_BaseNumericTransformer import (
+    BaseNumericTransformerInitTests,
+    BaseNumericTransformerTransformTests,
+)
 from tubular.numeric import InteractionTransformer
 
 
-class TestInit:
+class TestInit(BaseNumericTransformerInitTests):
     """Tests for InteractionTransformer.init()."""
 
-    def test_super_init_called(self, mocker):
-        """Test that init calls BaseTransformer.init."""
-        expected_call_args = {
-            0: {
-                "args": (),
-                "kwargs": {"columns": ["b", "c"], "verbose": True},
-            },
-        }
-
-        with ta.functions.assert_function_call(
-            mocker,
-            tubular.base.BaseTransformer,
-            "__init__",
-            expected_call_args,
-        ), contextlib.suppress(AttributeError):
-            InteractionTransformer(
-                columns=["b", "c"],
-                min_degree=2,
-                max_degree=2,
-                verbose=True,
-            )
+    @classmethod
+    def setup_class(cls):
+        cls.transformer_name = "InteractionTransformer"
 
     def test_invalid_input_type_errors(self):
         """Test that an exceptions are raised for invalid input types."""
-        with pytest.raises(
```

Comment:
Q: Do we not still need these? We've kept one of the checks in 'test_invalid_input_type_errors' and can't see that the other two are covered elsewhere?

Reply:
I thinkk these will be covered by GenericInitTests right as that looks at the columns arg?